### ISSUE TITLE
Can not modify the selected model

### ISSUE THREAD

**Issue by jaweber1** [FrSky-OS/discussion#34]
Not sure why it's blocked, but you can not modify the name or image for a model if it's been selected.

**Comment by janekx** [comment]
It is Ok that is no bug.

**Comment by jaweber1** [comment]
It may not be a bug, but I think it is something people will want to do.

**Comment by janekx** [comment]
You cant change basic setting when you have them in use.

**Comment by AloftHobbies** [comment]
I disagree. I know some manufacturers would agree with you, but why is this the case? I should be able to modify it all I like. If I am about to make a major change that will spill into other areas, then pop up a warning and ask me to confirm. 

Also, if something is not available to do, how about greying out the item in the menu. Make it so we can not select it. 

When in doubt avoid confusion.

**Comment by janekx** [comment]
Becuase the model is loaded in memory and model is name and template and image.  I can understand change image and name,  tamplate very hardly.

**Comment by janekx** [comment]
Is question how does FrSky OS works,  it needs FrSky to tell

**Comment by AloftHobbies** [comment]
Yes, but even if that is the case, they should be smart enough to write out and reload the model after modification.
